views/templates/hook/front/carrier-phone-number.tpl

```diff
@@ -8,18 +8,16 @@
                     class="chosen-select form-control form-control-chosen"
                     name="dpd-phone-area"
                     required="required">
-                    {html_options options=$dpdPhoneArea selected=$contextPrefix}
+                {html_options options=$dpdPhoneArea selected=$contextPrefix}
```

Comment:
As far I can see initially you leave it required, which means if default carrier is non DPD and customer wont trigger carrier change, we will still have a problem? Have you tested all cases?